docs/index.html

```diff
@@ -141,369 +141,326 @@ <h2 class="section-title">Schedule</h2>
-            <td class="schedule-slot"><span class="speaker-photo"><img class="photo" src="img/your-logo.jpg" alt="Registration &amp; Breakfast"></span></td>
-            <td class="schedule-description">Registration &amp; Breakfast</td>
-          </tr>
-        
       
-        
-          <tr class="schedule-other">
-            <td class="schedule-time">09:30 09:45</td>
-            <td class="schedule-slot"><span class="speaker-photo"><img class="photo" src="img/your-logo.jpg" alt="Opening Words"></span></td>
-            <td class="schedule-description">Opening Words</td>
-          </tr>
-        
+        <tr>
+          <td class="schedule-time">08:30 09:30</td>
+          <td class="schedule-slot">Registration &amp; Breakfast</td>
+          <td class="schedule-description"></td>
+        </tr>
+      
+      
+      
+      
+        <tr>
+          <td class="schedule-time">09:30 09:45</td>
+          <td class="schedule-slot">Opening Words</td>
+          <td class="schedule-description"></td>
+        </tr>
+      
       
-        
-          <tr class="schedule-other">
-            <td class="schedule-time">09:45 10:30</td>
-            <td class="schedule-slot"><span class="speaker-photo"><img class="photo" src="img/your-logo.jpg" alt="Keynote - Derek Parkar (author of the Delve Debugger)"></span></td>
-            <td class="schedule-description">Keynote - Derek Parkar (author of the Delve Debugger)</td>
-          </tr>
-        
       
-        
-          <tr class="schedule-other">
-            <td class="schedule-time">10:30 11:00</td>
-            <td class="schedule-slot"><span class="speaker-photo"><img class="photo" src="img/your-logo.jpg" alt="Break"></span></td>
-            <td class="schedule-description">Break</td>
-          </tr>
-        
       
-        
-          <tr>
-            <td class="schedule-time">11:00 11:30</td>
-            <td class="schedule-slot">
-            
-
-              Generics are a Gopher&#39;s Best Friend
-              <span class="speakers-company"></span>
-            </td>
-            <td class="schedule-description">Amit Lichtenberg
-              
-              
-              <img src="img/il.svg" style="float: right;width: 10px;height: 10px">
-              
-            </td>
-          </tr>
-        
       
         
-          <tr>
-            <td class="schedule-time">11:00 11:30</td>
-            <td class="schedule-slot">
-            
-
-              Shifting Left with Protobuf - a multi-disciplinary approach to schema management
-              <span class="speakers-company"></span>
-            </td>
-            <td class="schedule-description">Nir Barak
+        
+            <tr>
               
+              <td rowspan=1 class="schedule-time">09:45 10:30</td>
+                
+              <td class="schedule-slot"><span style="color: blue;">Derek Parkar-</span> Keynote</td>
+              <td class="schedule-description">author of the Delve Debugger
```

Comment:
Probably should be capital A
```suggestion
              <td class="schedule-description">Author of the Delve Debugger
```